spec/services/search/location_builder_spec.rb

```diff
@@ -13,7 +13,7 @@
 
   let(:location) { nil }
   let(:point_location) { "SW1A 1AA" }
-  let(:radius) { nil }
+  let(:radius) { 5 }
```

Comment:
For the tests of `buffer_radius`, this is making a `Search::LocationBuilder` that has both a radius and a buffer_radius parameter, which is a scenario we don't expect to happen.